### PR TITLE
AO3-5023 Fix Show Most Recent Bookmarks

### PR DESCRIPTION
**Pull request by sarken**
## Issue

https://otwarchive.atlassian.net/browse/AO3-5023

Specifically: https://otwarchive.atlassian.net/browse/AO3-5023?focusedCommentId=350605&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-350605

## Purpose

Fixes the Show Most Recent Bookmarks option

## Testing

Refer to JIRA

### CODE REVIEW

**Comment by zz9pzza** [comment]
@sarken looks like the travis failures are relevant.

**Comment by sarken** [comment]
I just realized you're getting an email every time I push. /o\ Closing for now. It might come back minus a test. :|

File: features/step_definitions/bookmark_steps.rb [hunk 1]
```diff
@@ -142,6 +141,13 @@
   click_button "Edit"
 end
 
+When /^I show most recent bookmarks on ([^"]*)'s bookmark of "([^"]*)"$/ do |user, title|
+  pseud_id = User.find_by(login: user).default_pseud.id
```
Comment (redsummernight) [inline features/step_definitions/bookmark_steps.rb]: You can avoid having to lookup the bookmark ID by grabbing the list element containing both the work title and the bookmarker:

```rb
  within("li.bookmark", text: /#{title}.*Bookmarked by #{user}/) do
    step %{I follow "Show Most Recent Bookmarks"}
  end
```

**Comment by redsummernight** [comment]
Looks good to me. :+1:

**Pull request merged**